best-practices/README.md

```diff
@@ -187,6 +187,14 @@ Email
 [MailView]: https://github.com/37signals/mail_view
 [ActionMailer Preview]: http://api.rubyonrails.org/v4.1.0/classes/ActionMailer/Base.html#class-ActionMailer::Base-label-Previewing+emails
 
+Web
+---
+
+* Prefer local assets to externally linked assets.
```

Comment:
How does this relate to cdn's? Those are external assets but generally preferred